base/applications/calc/lang/pt-BR.rc

```diff
@@ -1,6 +1,7 @@
 /* Translation and UTF-8 Conversion by mkbu95 <mkbu95@gmail.com> (August, 2011) */
+/* Modifications by Gabriel Aguiar fgygh5804@gmail.com> (3, August, 2021) */
```

Comment:
One translator is enough per file. First come, first served. Delete that added line therefore.

Reply:
We actually have multiple translators in translation files @JoachimHenze